web/modules/custom/ecc_cookie_compliance/ecc_cookie_compliance.libraries.yml

```diff
@@ -10,3 +10,4 @@ live_handler:
     js/live_handler.js: {}
   dependencies:
     - core/jquery
+    - eu_cookie_compliance
```

Comment:
Are you sure this is needed? If you're specifying that this _module_ is depended on the `eu_cookie_compliance` module that's already been done in `ecc_cookie_compliance.info.yml`.